### DISCUSSION TITLE
Generate a 2d model of a semicircle

### DISCUSSION THREAD

**Discussion by bright-men** [neperfepx/neper#804] [Q&A]
We can generate two-dimensional circles using Neper, and there are many applications in three-dimensional cutting. But I do not know what command should be used to cut a circle to generate a semi-circle.

**Comment by rquey** [comment]
See `split` in [-domain](https://neper.info/doc/neper_t.html#cmdoption-domain).  Here is an example:

```
neper -T -domain "circle(1):split(x)"  -n 100 -dim 2
neper -V n100-id1.tess -print img
```
![img](https://github.com/neperfepx/neper/assets/2659376/c9c87015-2ad8-4c84-ae86-566d2264b796)

**Reply by bright-men** [reply]
thank you very much
